Yosemite/YosemiteTests/Stores/AppSettingsStoreTests+ProductsFeatureSwitch.swift

```diff
@@ -21,12 +21,24 @@ final class AppSettingsStoreTests_ProductsFeatureSwitch: XCTestCase {
     ///
     private var subject: AppSettingsStore!
 
+    // Previous feature switch file URL
+    private lazy var productsFeatureSwitchURL: URL = {
+        let documents = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first
+        return documents!.appendingPathComponent("products-feature-switch.plist")
+    }()
```

Comment:
Should we still keep this file since it is no longer used?

Reply:
it's still used when deleting feature switches - since it's possible the device had the previous version with the Products M2 feature switch PList file, I thought it'd be nice to delete all previous feature switches here. Lemme know if you think there's a better way to clean up the previous feature switch file!

Reply:
Thanks for clarifying this aspect. The only thing is that probably if we remove it in the future, we will always have users with this plist file stored, maybe because they are updating from an old version to a recent version of the app. In short I think we will never be able to completely remove all the old plist files unless we keep track of them somewhere forever.